SQL/updates/28-29.sql

```diff
@@ -0,0 +1,4 @@
+# Updates DB from 28 to 29 -S34N_W
+# Adds support for screentips
+ALTER TABLE `player` ADD COLUMN `screentip_mode` varchar(4) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '8px';
```

Comment:
```suggestion
ALTER TABLE `player` ADD COLUMN `screentip_mode` varchar(4) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '8px';
```
Make this column a `TINYINT` (Stores -128 to 127 or 0-255 depending on if you sign it or not) and avoid storing `px` in the database.

Make the size thing a user entry box as well so they can pick a custom size instead of 0/8//11/15.

Reply:
I can definitely do a user entry box, but is that wise? Most people are dumb as a pile of rocks, giving them a "small/medium/large/off" is probably better from a UX design PoV, right?

Reply:
As for the rest, I can do that no worries

Reply:
`Please enter a tooltip size. (We recommend something between 8 & 15)`